homeassistant/components/fitbit/const.py

```diff
@@ -64,198 +64,231 @@ class FitbitSensorEntityDescription(SensorEntityDescription, FitbitRequiredKeysM
         key="activities/tracker/calories",
         name="Tracker Calories",
         unit_type="cal",
         icon="mdi:fire",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/distance",
         name="Tracker Distance",
         unit_type="",
         icon="mdi:map-marker",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/elevation",
         name="Tracker Elevation",
         unit_type="",
         icon="mdi:walk",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/floors",
         name="Tracker Floors",
         unit_type="floors",
         icon="mdi:walk",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/minutesFairlyActive",
         name="Tracker Minutes Fairly Active",
         unit_type=TIME_MINUTES,
         icon="mdi:walk",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/minutesLightlyActive",
         name="Tracker Minutes Lightly Active",
         unit_type=TIME_MINUTES,
         icon="mdi:walk",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/minutesSedentary",
         name="Tracker Minutes Sedentary",
         unit_type=TIME_MINUTES,
         icon="mdi:seat-recline-normal",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/minutesVeryActive",
         name="Tracker Minutes Very Active",
         unit_type=TIME_MINUTES,
         icon="mdi:run",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/steps",
         name="Tracker Steps",
         unit_type="steps",
         icon="mdi:walk",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="body/bmi",
         name="BMI",
         unit_type="BMI",
         icon="mdi:human",
+        state_class="measurement",
     ),
     FitbitSensorEntityDescription(
         key="body/fat",
         name="Body Fat",
         unit_type=PERCENTAGE,
         icon="mdi:human",
+        state_class="measurement",
     ),
     FitbitSensorEntityDescription(
         key="body/weight",
         name="Weight",
         unit_type="",
         icon="mdi:human",
+        state_class="measurement",
     ),
     FitbitSensorEntityDescription(
         key="sleep/awakeningsCount",
         name="Awakenings Count",
         unit_type="times awaken",
         icon="mdi:sleep",
+        state_class="total_increasing",
     ),
     FitbitSensorEntityDescription(
         key="sleep/efficiency",
         name="Sleep Efficiency",
         unit_type=PERCENTAGE,
         icon="mdi:sleep",
+        state_class="total_increasing",
```

Comment:
I don't think that the state class `total_increasing` makes sense for percentage values. Shouldn't it be measurement?

Reply:
Thank you. That's right